pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+env =
+    # System fonts can interfere with screenshots that expect a particular font
+    # to be used
+    PYGFX_DISABLE_SYSTEM_FONTS=1
```

Comment:
I guess you are defining this here since you also want it to be set when running tests locally, and not just on CI.

Reply:
yes. though I can see how a test that checks system font integration might be a good idea.... I just think this is more worth it.

Reply:
(I'm hesitant to resolve discussions myself. I presonally don't like the interaction. Its like you (Korjin) asked a question, and I get to decide if it was answered in a satisfactory way? Then I collapse the entire discussion to make you forget that you had a question in the first place....)